tests/cbmc/proofs/Makefile.common

```diff
@@ -258,6 +258,7 @@ CBMC_FLAG_FLUSH ?= --flush
 # CBMC flags used for property checking and coverage checking
 
 CBMCFLAGS += $(CBMC_FLAG_FLUSH)
+COVERFLAGS ?= --no-standard-checks
```

Comment:
May be worth adding a link to https://github.com/diffblue/cbmc/issues/8389: we will likely issue a fix so that with future CBMC versions this won't be needed anymore.

Reply:
Noted and will do.